.gitignore

```diff
@@ -18,3 +18,9 @@ erl_crash.dump
 
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
+
+# Ignore ElixirLS directory
+/.elixir_ls
+
+# Ignore MacOS
+.DS_Store
```

Comment:
This change is not project-specific but depends on a development environment. It's better to add those to your [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).